bluetooth/README.md

```diff
@@ -67,3 +92,8 @@ class CommonViewModel(bltBuilder: BluetoothBuilder) : BaseViewModel() {
     )
 } 
 ```
+
+### Notes
+There is a major difference when it comes to default reported device emissions between Android and iOS. Android will report multiple emissions of the same device, as iOS will filter them out.
+
+To align the behaviour across platform the [CBCentralManagerScanOptionAllowDuplicatesKey](https://developer.apple.com/documentation/corebluetooth/cbcentralmanagerscanoptionallowduplicateskey) option is enabled on iOS.
```

Comment:
```suggestion
To align the behaviour across platforms the [CBCentralManagerScanOptionAllowDuplicatesKey](https://developer.apple.com/documentation/corebluetooth/cbcentralmanagerscanoptionallowduplicateskey) option is enabled on iOS. It can be set to another value using `ScanSettings` as shown above.
```